tests/test_config.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+
+cwd = os.getcwd()
+if cwd not in {'root/project', '/opt/code/tap-square'}:
+    print("WARN: This script is meant to run from the top level directory of the tap.")
+
+print("Reading in filenames from tests directory.")
+print("Parsing directory for tests.")
```

Comment:
You print these two, but can you print the files found in a line below?

Reply:
Done